cdap-docs/admin-manual/source/security/authorization.rst

```diff
@@ -354,97 +347,102 @@ Kerberos Principal
 
 .. _security-pre-grant-wildcard-privilege:
 
-Pre-grant and Wildcard Privileges
-=================================
-The new authorization model requires pre-granted privilege on all entity for any operation.
-When CDAP is first started with authorization enabled, no users are granted privileges on
-any CDAP entities. Without any privileges, CDAP will not be able to create the default namespace.
-To create the default namespace, grant *ADMIN* on default namespace to the CDAP master user.
-The default namespace will get created in several minutes automatically.
+Wildcard Privileges
+===================
+Wildcard privileges can be used to simplify granting privileges on multiple entities.
+Wildcards can be used in the entity name to grant or revoke actions on multiple entities.
+
+- ``*`` matches zero or more characters
+- ``?`` matches a single character
 
-To pre-grant the privilege, wildcard can be used to minimize the burden of granting privileges on all entities.
-Detailed ways of granting privileges can be found in the following sections for different authorization backends.
+The following sections provide examples on granting wildcard privileges.
 
 .. _security-sentry-integration:
 
 Sentry Integration
 ------------------
 :ref:`CDAP CLI <cdap-cli>` can be used to grant or revoke the privileges for :ref:`Integrations: Apache Sentry <apache-sentry>`.
+Full list of commands are documented at :ref:`security commands <cli-available-commands-security>`.
 
-You can use the :ref:`CDAP CLI <cdap-cli>` to issue :ref:`security commands <cli-available-commands-security>`.
-Wildcard can be used to grant or revoke actions on multiple entities by including ``*`` and ``?`` in the entity name:
+Sentry only allows granting privileges to roles. Roles can then be assigned to groups.
 
-- To grant a principal privileges to perform certain actions on an entity, use::
-
-    > grant actions <actions> on entity <entity-id> to <principal-type> <principal-name>
-    > revoke actions <actions> on entity <entity-id> from <principal-type> <principal-name>
-
-  where:
-
-  - ``<actions>`` is a comma-separated list of privileges, any of *READ, WRITE, EXECUTE,* or *ADMIN*.
+- To create a new role, use::
 
-  - ``<entity>`` is of the form ``<entity-type>:<entity-id>``, where ``<entity-type>`` is
-    one of ``namespace``, ``artifact``, ``application``, ``dataset``, ``program``, ``stream``, ``dataset_type`` or
-    ``dataset_module``.
+    > create role <role-name>
 
-  - For namespaces, ``<entity-id>`` is composed from the namespace, such as
-    ``namespace:<namespace-name>``.
+- To grant a role privileges to perform certain actions on an entity, use::
```

Comment:
Should we say `To grant or revoke` here since we also provide revoke command below

Reply:
Done